src/dowel/csv_output.py

```diff
@@ -56,6 +54,36 @@ def record(self, data, prefix=''):
         else:
             raise ValueError('Unacceptable type.')
 
+    def _augment(self, data):
+        """Augment tabular data with new column(s)"""
+        data.reset()
```

Comment:
Why is this call to `reset()` necessary? Please add a comment.

Reply:
Good advice!